### DISCUSSION TITLE
underworld rule clarification?

### DISCUSSION THREAD

**Discussion by phillipfung3** [bafolts/terraforming-mars#6768] [General]
played underworld with turmoil in, current global event is volcanic eruption
excavated one of those 2 steel per temperature thingy within this turn, and find that it didn't trigger on the global event
just wondering is this how it is supposed to be?
by my understanding, resolving global event is part of the generation, so I thought it would work

**Comment by kberg** [comment]
Why would an excavation trigger as part of a global event? Excavation does not occur except during a player action.

**Reply by phillipfung3** [reply]
these thingys, according to rulebook they last till the end of gen, and it works like magic in a game where we pushed the temperature bars by 12 steps in one gen :)
![image](https://github.com/terraforming-mars/terraforming-mars/assets/172607360/ba320833-7a23-45c0-8a2e-1c7d336ae23d)

and its chemistry with this event
![image](https://github.com/terraforming-mars/terraforming-mars/assets/172607360/a1275d6f-ebf7-4136-9605-6d1c12244c5d)

**Reply by kberg** [reply]
Ah, that's a good question. I assume no, but am asking. Follow here: https://boardgamegeek.com/thread/2907313/article/44489109#44489109

**Comment by kberg** [comment]
Tracking this as a bug

**Comment by kberg** [comment]
FYI this is being fixed.
